index.html

```diff
@@ -27,15 +27,15 @@
           </aside>
         <div class="productContainer">
           <div class="productWrapper" >
-            <img class="productImage" src="pictures/backwardsWateringCan.jpg"/>
+            <img class="productImage" id="left" src="pictures/backwardsWateringCan.jpg" tabindex="1"/>
             <button class="selecter" id="button1" type="button">Select</button>
           </div>
           <div class="productWrapper" >
-            <img class="productImage" src="pictures/iceCreamLock.jpg"/>
+            <div id="mid" tabindex="1"><!--<img class="productImage" src="pictures/iceCreamLock.jpg" />--></div>
             <button class="selecter" id="button2" type="button">Select</button>
           </div>
           <div class="productWrapper" >
-            <img class="productImage" src="pictures/pizzaPouch.jpg"/>
+            <img id="right" class="productImage" src="pictures/pizzaPouch.jpg" tabindex="1"/>
```

Comment:
I remain impressed that you figured out that `tabindex` will let non-form elements be styled using `:focus`.